examples/02-HFSS/HFSS_FSS_unitcell.py

```diff
@@ -0,0 +1,229 @@
+"""
+HFSS: dipole antenna
```

Comment:
@pvinod13 I think the title is wrong

Reply:
Title has been changed in the commit #3655